spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/ChannelBindingServiceConfiguration.java

```diff
@@ -64,10 +65,10 @@
 	// it is used to detect a ChannelBindingService in the parent context (which we know
 	// already exists).
 	@ConditionalOnMissingBean(ChannelBindingService.class)
-	public ChannelBindingService bindingService(
-			ChannelBindingServiceProperties channelBindingServiceProperties,
+	public ChannelBindingService bindingService(ChannelBindingServiceProperties channelBindingServiceProperties,
 			Binder<MessageChannel> binder) {
-		return new ChannelBindingService(channelBindingServiceProperties, binder);
+		return new ChannelBindingService(channelBindingServiceProperties, binder,
+				messageConvertersConfigurer(channelBindingServiceProperties));
```

Comment:
I think we can rather pass the instance as an argument to the `bindingService()` method instead.